src/plugins/flow.js

```diff
@@ -878,11 +878,11 @@ export default function (instance) {
   });
 
   instance.extend("toAssignable", function (inner) {
-    return function (node) {
+    return function (node, isBinding) {
```

Comment:
That's what I originally had tbh, then noticed `toAssignableList` was being explicit with the args.

Shall I change it?